admob/build.gradle.kts

```diff
@@ -49,7 +49,7 @@ android {
 }
 
 dependencies {
-    implementation("org.godotengine:godot:$godotVersion.stable")
+    implementation("org.godotengine:godot:4.3.0.stable")
```

Comment:
Why did we hardcode the godot version value here rather than taking it from `godotVersion` constant?